client/src/components/Login/LoginContainer.tsx

```diff
@@ -1,51 +1,29 @@
-import React from 'react';
-import {connect} from 'react-redux';
+import React from "react";
 import Login from "./Login";
-import {setEmailText, setPasswordText, getApiData} from "../../store/login/actions";
-
-interface FormProps {
-    email: string;
-    password: string;
-    token: any;
-    setEmailText: Function;
-    setPasswordText: Function;
-    getApiData: Function;
-}
-
-interface LoginState {
-    email: string;
-    password: string;
-    token: any;
-}
-
-
-interface FormState {
-    login: LoginState;
-}
-
-class LoginContainer extends React.Component<FormProps, FormState> {
-    render() {
-        return <Login email={this.props.email}
-                      password={this.props.password}
-                      setEmailText={this.props.setEmailText}
-                      setPasswordText={this.props.setPasswordText}
-                      getApiData={this.props.getApiData}
-        />;
-    }
+import { reduxForm } from "redux-form";
+import API from "../../store/api";
+
+const TOKEN = "TOKEN";
+
+class LoginContainer extends React.Component<any> {
+  onSubmit = (user: any) => {
+    return API.post("/auth/login", user)
```

Comment:
move to action and use props, don't forgot about pending, success, and error handling